### PR TITLE
Add cypress github action

### LINKED ISSUE #274: Configure Cypress with Github Actions
## Is your feature request related to a problem?

*Please describe.*

## Describe the solution you'd like
There is a CI solution for the Cypress in [Github Actions](https://docs.cypress.io/guides/continuous-integration/github-actions?utm_source=Test%20Runner&utm_medium=CI%20Prompt%201&utm_campaign=GitHub&utm_content=Automatic#Cypress-GitHub-Action)

### PR DESCRIPTION
**Pull request by aggmoulik**
Resolves #274 

Signed-off-by: Moulik Aggarwal <qwertymoulik@gmail.com>

### CODE REVIEW

**Comment by palash-signoz** [comment]
Hi @aggmoulik 
Thanks for contributing will your PR

**Comment by aggmoulik** [comment]
I have tried the env just like in the actions documentation but it is not working for some reason.

**Comment by aggmoulik** [comment]
Hey @palash-signoz, You can refer [here](https://github.com/aggmoulik/signoz/pull/1) to see why tests are not working.
If I am missing something let me know.

**Comment by pal-sig** [comment]
Hi @aggmoulik 
I am getting time out error

File: .github/workflows/cypress.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: run cypress e2e tests
+        uses: cypress-io/github-action@v2.9.7
+        with:
+          start: yarn dev
```
Comment (pal-sig) [inline .github/workflows/cypress.yaml:13]: @aggmoulik i think dev server start with 3000 port

File: .github/workflows/cypress.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          wait-on: http://localhost:8080/
+          config-file: cypress.json
+          working-directory: frontend
+          headless: true
+          browser: chrome
+        env:
+          CYPRESS_baseUrl: http://localhost:8080/
```
Comment (pal-sig) [inline .github/workflows/cypress.yaml:20]: can you use env file for same ?

**Review by pal-sig — CHANGES_REQUESTED** [review]

**Comment by palash-signoz** [comment]
@aggmoulik any update regarding the changes ?

**Comment by pranay01** [comment]
Stale PR. Closing

**Pull request closed without merging**